### ISSUE TITLE
Universal Package from Azure

### ISSUE THREAD

**Issue by wldevries** [Inedo/Inedo.UPack#31]
https://devblogs.microsoft.com/devops/getting-started-with-universal-packages/

It seems Azure also has a concept called Universal Packages. Are the two related and should a distinction be made in Inedo documentation?

**Comment by apxltd** [comment]
Funny thing... Azure DevOps' Universal Packages came out about two years after we launched Universal Packages - and they even shared *word-for-word* the same texts/descriptions! Just like how other ADO Packages features shared the same text/descriptions as the competitor's. But I guess great minds alike 🤣

Now if only they copied the specifications like virtual packages, extensible metadata, etc... they'd be useful. They're just files with no real metadata that I can tell. Name and version I guess? 

But sadly our SEO juice isn't nearly as good as Microsoft's so Google will sometimes favor them in a search 🙄

Long story short, there's no relation and I'm still a little salty.
